packages/ses/src/module-instance.js

```diff
@@ -158,6 +165,14 @@ export const makeModuleInstance = (
   // both initialize and update live bindings.
   const liveVar = create(null);
 
+  const importMeta = create(null);
+  if (moduleRecordMeta) {
+    assign(importMeta, moduleRecordMeta);
+  }
+  if (usesImportMeta && importMetaHook) {
+    importMetaHook(importMeta);
```

Comment:
We probably won’t regret making this `importMetaHook(moduleSpecifier, importMeta)`